packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -48,24 +56,30 @@ const defaultProps = {
   isTruncated: false,
   accessoryAlignment: 'alignCenter',
   startAccessory: undefined,
+  reserveStartAccessorySpace: false,
   endAccessory: undefined,
+  reserveEndAccessorySpace: false,
   displays: [],
   comment: undefined,
 };
 
 class ItemView extends React.Component {
 
-  static renderAccessory(accessory, accessoryAlignment) {
+  static renderAccessory(accessory, reserveSpace, accessoryAlignment, type) {
     let accessorySection;
-    if (accessory) {
+    if (accessory || reserveSpace) {
+      const reserveSpaceClassName = reserveSpace ? `reserve-${type}-accessory-space` : null;
```

Comment:
do we really want null to be in the classname?  wouldn't an empty string '' be better?

Reply:
nope we def don't